masterlist.yaml

```diff
@@ -2194,6 +2194,26 @@ plugins:
         subs: [ 'Previsibines Repair Pack v0.57+' ]
         condition: 'active("PPF.esm")'
 
+  - name: 'FO4LODGen(-|-DLC)?(Coast-WindTurbines|Coast|FullModelLOD|HighTrees|NukaWorld|Workshop03)?\.esp'
+    url:
+      - link: 'https://stepmodifications.org/forum/topic/17508-fo4lodgen/'
+        name: 'FO4LODGen Resources'
+    group: *fixesGroup
+  - name: 'FO4LODGen(-|-DLC)?(Coast|FullModelLOD|HighTrees|NukaWorld|Workshop03)?\.esp'
+    msg:
+      - type: say
+        content: 'This plugin may be disabled after generating LOD with xLODGen.'
```

Comment:
@DoubleYouC 
I'll assume that you've looked at our existing `&runxLODGen` message anchor and concluded, that it doesn't really fit in this situation, right?

We could turn this into a new message anchor, like so:
```yaml
  - &disableAfterxLODGen
    type: say
    content:
      - lang: en
        text: 'This plugin may be disabled after generating LOD with **xLODGen**.'
```
@sibir-ine
What's your opinion on this?

Reply:
The existing &runxLODGen message only applies to Occlusion.esp as generated by xLODGen for older games for Occlusion data generated in the cell records, preventing lod blocks from occluding incorrectly. Fallout 4 uses Previs for occlusion and xLODGen has the Occlusion option disabled. 

Fallout 4 has a bug with statics having HasDistantLOD flag that are also craftable in settlements. It will briefly display all lod if you select a craftable object with the flag, which is probably why Bethesda didn't add lod to many static records that Sheson has added. Since it is only necessary to have it flagged as such at lod generation time, as noted by Sheson in his Readme and verified in my own testing, I added this note.

Now this note does apply to two additional plugins from two other mods, so if we want to add it to common, I suppose we could, but I couldn't seem to figure out how that worked.

Reply:
The process is to first introduce the new message anchor in prelude.yaml. I'm gonna do this now - I'm also gonna change it slightly from my previous suggestion to be more generalised:
```yaml
  - &disableAfterGeneratingXwithY
    type: say
    content:
      - lang: en
        text: 'This plugin may be disabled after generating **%1%** with **%2%**.'
```
Once that is done, I'm also gonna add a short definition of it (only the english sentence) to the `prelude/common` node within each masterlist. These short definitions are used, so that masterlist maintainers and contributors can easily see, which message anchors are available and thus help them in deciding, which message is the best for each individual use case.

After that, in the masterlist where it is needed, so here Fallout 4, I'm gonna add a sub-anchor in the `common` node:
```yaml
  # &disableAfterGeneratingXwithY
  - &disableAfterGeneratingLODwithxLODGen
    <<: *disableAfterGeneratingXwithY
    subs:
      - 'LOD'
      - 'xLODGen'
```
Once I'm done, you can use this new sub-anchor, so:
```yaml
  - name: 'FO4LODGen(-|-DLC)?(Coast|FullModelLOD|HighTrees|NukaWorld|Workshop03)?\.esp'
    msg: [ *disableAfterGeneratingLODwithxLODGen ]
```

Reply:
Done. You may need to rebase so that you can use the new sub-anchor.

Reply:
Rebased and changed to use the new sub-anchor.